7001-8000.po

```diff
@@ -301,22 +301,22 @@ msgstr "Duplikant nie mógł znaleźć toalety i jest bardzo zawstydzony"
 #. STRINGS.DUPLICANTS.STATUSITEMS.STRESSVOMITING.NAME
 msgctxt "STRINGS.DUPLICANTS.STATUSITEMS.STRESSVOMITING.NAME"
 msgid "Stress vomiting"
-msgstr "Stres wymiotny"
+msgstr "Wymioty ze stresu"
 
 #. STRINGS.DUPLICANTS.STATUSITEMS.STRESSVOMITING.NOTIFICATION_NAME
 msgctxt "STRINGS.DUPLICANTS.STATUSITEMS.STRESSVOMITING.NOTIFICATION_NAME"
 msgid "Stress vomiting"
-msgstr "Stres wymiotny"
+msgstr "Wymioty ze stresu"
 
 #. STRINGS.DUPLICANTS.STATUSITEMS.STRESSVOMITING.NOTIFICATION_TOOLTIP
 msgctxt "STRINGS.DUPLICANTS.STATUSITEMS.STRESSVOMITING.NOTIFICATION_TOOLTIP"
 msgid "These Duplicants became too <style=\"stress\">Stress</style> and it caused them to throw up:"
-msgstr ""
+msgstr "Duplikanci są za bardzo <style=\"stress\">Zestresowani</style> i przez to wymiotują: "

```

Comment:
W sumie racja ;) Choć raczej jest to logiczne, że Ci a nie tamci których to nie tyczy ;) Dopiszę, kein problem ;)